octopoes/octopoes/core/service.py

```diff
@@ -154,7 +165,7 @@ def save_origin(self, origin: Origin, oois: list[OOI], valid_time: datetime) ->
             try:
                 self.ooi_repository.get(origin.source, valid_time)
             except ObjectNotFoundException:
-                return
+                raise ValueError("Origin source of observation does not exist")
```

Comment:
I spotted this while debugging what went wrong in saving. I think we should never silently not save something when the data is not correct, but always return an error...